bokehjs/src/less/main.less

```diff
@@ -153,11 +11,3 @@
   border: 1px dashed green;
   z-index: 100;
 }
-
-.bk-toolbar-button.hover:focus {
-  outline: none;
-}
-
-.box .bk-grid-column {
```

Comment:
I can't think of one - fine with me that it's gone. Sorry for the v. late reply.